Branch-SDK/Branch-SDK/Branch.h

```diff
@@ -436,6 +436,8 @@ typedef NS_ENUM(NSUInteger, BranchPromoCodeUsageType) {
  */
 - (void)setRequestMetadataKey:(NSString *)key value:(NSObject *)value;
 
+- (void)setThirdPartyUniversalLinksRegex:(NSRegularExpression *)regex;
```

Comment:
Given that ESPs are a dime-a-dozen, I think we should make this an app config and return it in /v1/open.